packages/app/pages/organization/course/[cid]/edit.tsx

```diff
@@ -214,14 +215,27 @@ export default function Edit(): ReactElement {
                 }}
                 onFinish={updateGeneral}
               >
+                {courseData.profIds.length !== professors?.length && (
+                  <Alert
+                    message="System Notice"
+                    description="One or more professors in this organization is not assigned a professor role. Please assign a professor role to all professors in this course."
+                    type="error"
+                    style={{ marginBottom: 20 }}
+                  />
+                )}
```

Comment:
Bug: This seems to lock me out of editing anything on my newly created course.

Steps to reproduce:
1. Create a course with Ramon Lawrence as a professor
2. Click edit course (warnings should show up)

Reply:
After looking at this file, it seems like `professors` is the total professors for the *organization*, not the course. So it seems like doing `courseData.profIds.length !== professors?.length` is comparing the total professors in an *organization* and the number of professors in the *course* and seeing if they are the same (and they basically never would be)

Lines 81-84: 
```ts
 const { data: professors } = useSWR(
      isAdmin ? `/api/v1/organization/[oid]/get_professors` : null,
      async () => await API.organizations.getProfessors(organization.id),
    )
```

Reply:
Okay, I will remove this check since I asked Ramon to update Scott's role to prof

Reply:
Reverted